src/js/main.js

```diff
@@ -0,0 +1,39 @@
+function updateFilms(films)
+{
+    // clear
+    let clearFilms = document.querySelectorAll('.sidebar__item')
+    for (let i = 0; i < clearFilms.length; i++)
+    {
+        clearFilms[i].remove()
+    }
+    // show
+    let sidebar_items = document.querySelector('.sidebar__items')
+    films.forEach((film) => {
+        const li = document.createElement('li')
+        li.className = 'sidebar__item'          // можно classList
+        li.append(film)                         // можно innerHTML
+        sidebar_items.append(li)
+    })
+}
+
+
+document.querySelector('.sidebar__header').innerHTML = 'Фильмы'
+const films = ["Зелёная книга", "1 + 1", "Плохие парни", "Молодость", "Мы - Миллеры", 
+               "Однажды в Голливуде", "Дэдпул", "Копы в юбках", "Шпион"]
+
+updateFilms(films)
```

Comment:
```suggestion
const sidebar_items = document.querySelector(".sidebar__items");

const films = [
  "Зелёная книга",
  "1 + 1",
  "Плохие парни",
  "Молодость",
  "Мы - Миллеры",
  "Однажды в Голливуде",
  "Дэдпул",
  "Копы в юбках",
  "Шпион",
];

updateFilms(films);

function updateFilms(films) {
  sidebar_items.innerHTML = "";
  films.forEach((film) => {
    const li = document.createElement("li");
    li.className = "sidebar__item"; 
    li.append(film);
    sidebar_items.append(li);
  });
}
```

проще очищать контейнер через innerHTML, ну и заранее найти sidebar_items, чтоб при каждом поиске не искать его в dom

Reply:
а `document.querySelector(".sidebar__header").innerHTML = "Фильмы";`
можно же сразу настроить на фильмы